app/src/main/res/layout/activity_reservation.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/walpaper_reservation"
+    android:gravity="center_horizontal"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/title_activity_reservation"
+        style="@style/ActivityReservationTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content" />
+
+    <Button
+        android:id="@+id/button_initial_date_time_picker_activity_reservation"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:backgroundTint="@color/yellow"
+        android:text="@string/button_initial_date_time_picker_reservation" />
+
+    <Button
+        android:id="@+id/button_end_date_time_picker_activity_reservation"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:backgroundTint="@color/yellow"
+        android:text="@string/button_end_date_time_picker_reservation" />
+
+    <EditText
+        android:id="@+id/parking_place_activity_reservation"
+        style="@style/EditTextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/height_activity_reservation_edittext_box"
+        android:hint="@string/edit_text_parking_place"
+        android:imeOptions="actionDone"
+        android:inputType="number" />
+
+    <EditText
+        android:id="@+id/code_activity_reservation"
```

Comment:
please use the naming convention, <what>_<where>_<description>

Reply:
Done